recipes/openassetio/all/conanfile.py

```diff
@@ -0,0 +1,177 @@
+        return {
+            "gcc": "9",
+            "clang": "12",
+            "apple-clang": "12",
+        }
+
+    def configure(self):
+        if self.options.without_python:
+            self.options.rm_safe("python_version")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("tomlplusplus/3.2.0")
+        if not self.options.without_python:
+            self.requires("openssl/1.1.1s")
+            self.requires(f"cpython/{self.options.python_version}")
+            if is_msvc(self):
+                # Required to create import .lib for building extension module.
+                self.options["cpython"].shared = True
+            self.requires("pybind11/2.10.0")
+
+    def validate(self):
+        if is_apple_os(self):
+            raise ConanInvalidConfiguration(
+                f"{self.ref} does not support MacOS at this time"
+            )
+        if self.info.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        check_min_vs(self, 191)
+        if not is_msvc(self):
+            minimum_version = self._compilers_minimum_version.get(str(self.info.settings.compiler), False)
+            if minimum_version and Version(self.info.settings.compiler.version) < minimum_version:
+                raise ConanInvalidConfiguration(
+                    f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+                )
+
+    def build_requirements(self):
+        self.tool_requires("cmake/3.21.7")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+
+        tc.variables["OPENASSETIO_ENABLE_TESTS"] = not self.conf.get("tools.build:skip_test", default=True, check_type=bool)
+
+        libcxx = self.settings.get_safe("compiler.libcxx")
+        if libcxx is not None:
+            if libcxx == "libstdc++11":
+                tc.variables["OPENASSETIO_GLIBCXX_USE_CXX11_ABI"] = True
+            else:
+                tc.variables["OPENASSETIO_GLIBCXX_USE_CXX11_ABI"] = False
+
+        if self.options.without_python:
+            tc.variables["OPENASSETIO_ENABLE_PYTHON"] = False
+        else:
+            tc.variables["OPENASSETIO_ENABLE_PYTHON"] = True
+            tc.variables["Python_EXECUTABLE"] = self._python_exe
+            if is_msvc(self):
+                tc.variables["Python_LIBRARY"] = self._python_windows_lib
+
+        tc.generate()
+        tc = CMakeDeps(self)
+        tc.generate()
+        tc = VirtualBuildEnv(self)
+        tc.generate(scope="build")
+
+    @property
+    def _python_exe(self):
+        return pathlib.Path(self.deps_user_info["cpython"].python).as_posix()
+
+    @property
+    def _python_windows_lib(self):
+        pth = pathlib.Path(
+            self.deps_cpp_info["cpython"].rootpath,
```

Comment:
```suggestion
            self.dependencies["cpython"].package_folder,
```

Reply:
Nice! That seems to work, switched in 619402f3